HopsanGUI/LogDataHandler2.cpp

```diff
@@ -756,20 +756,45 @@ void LogDataHandler2::importFromCSV_AutoFormat(QString importFilePath)
     if (file.open(QFile::ReadOnly))
     {
         QTextStream ts(&file);
-        QStringList fields = ts.readLine().split(',');
-        if (!fields.isEmpty())
+        QStringList firstRow = ts.readLine().split(',');
+        QStringList firstColumn;
+        firstColumn.push_back(firstRow.first());
```

Comment:
That's true. I assumed the file was not empty. I will add a check.

Reply:
Maybe the error message should be adjusted, unlikely but if someone has a file with the first line blank, the message wont make sense. The best thing would be to read firstLine until its not empty.... but maybe it can wait since I am improving the csv parser library and intend to use it in the GUI also. I dont know I can handled this special case there either.

Reply:
I will change the error message to something like "First line is empty". Of course we could skip the first lines if they are empty, but that's not really a necessary feature. Like you write below, we must assume that the files are valid somehow. We can simply decide that Hopsan will not support CSV files that starts with a blank line.